src/connectors/refinement-list/connectRefinementList.js

```diff
@@ -154,126 +154,62 @@ export default function connectRefinementList(renderFn, unmountFn = noop) {
     });
     const getLimit = isShowingMore => (isShowingMore ? showMoreLimit : limit);
 
-    let lastResultsFromMainSearch = [];
+    let lastResultsFromMainSearch;
+    let lastItemsFromMainSearch = [];
     let hasExhaustiveItems = true;
     let searchForFacetValues;
     let triggerRefine;
 
-    const render = ({
-      items,
-      state,
-      createURL,
-      helperSpecializedSearchFacetValues,
-      refine,
-      isFromSearch,
-      isFirstSearch,
-      isShowingMore,
-      toggleShowMore,
-      instantSearchInstance,
-    }) => {
-      // Compute a specific createURL method able to link to any facet value state change
-      const _createURL = facetValue =>
-        createURL(state.toggleRefinement(attribute, facetValue));
-
-      // Do not mistake searchForFacetValues and searchFacetValues which is the actual search
-      // function
-      const searchFacetValues =
-        helperSpecializedSearchFacetValues &&
-        helperSpecializedSearchFacetValues(
-          state,
-          createURL,
-          helperSpecializedSearchFacetValues,
-          refine,
-          instantSearchInstance,
-          isShowingMore
-        );
-
-      const canShowLess =
-        isShowingMore && lastResultsFromMainSearch.length > limit;
-      const canShowMore = showMore && !isFromSearch && !hasExhaustiveItems;
-
-      const canToggleShowMore = canShowLess || canShowMore;
-
-      renderFn(
-        {
-          createURL: _createURL,
-          items,
-          refine,
-          searchForItems: searchFacetValues,
-          instantSearchInstance,
-          isFromSearch,
-          canRefine: isFromSearch || items.length > 0,
-          widgetParams,
-          isShowingMore,
-          canToggleShowMore,
-          toggleShowMore,
-          hasExhaustiveItems,
-        },
-        isFirstSearch
-      );
-    };
-
     /* eslint-disable max-params */
```

Comment:
is this disable still useful? if it's not public api, could we make it take an object as arguments instead?

Reply:
Yep, it is not useful anymore, I removed it.